contracts/bases/QuoteVerifierBase.sol

```diff
@@ -143,7 +143,9 @@ abstract contract QuoteVerifierBase is IQuoteVerifier, EnclaveIdBase, X509ChainB
     function serializeOutput(Output memory output) internal pure returns (bytes memory) {
         return abi.encodePacked(
             output.quoteVersion, 
-            output.tee, 
```

Comment:
I think we can keep `tee` little-endian encoded, because the constant definition for TEE_TYPE is represented in LE as well.